### PR TITLE
Display version of SSH

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Add SSH version of the VM to the output.

---

**Related issue :** https://github.com/EGI-Federation/fedcloud-vm-monitoring/issues/15

### CODE REVIEW

File: fedcloud_vm_monitoring/site_monitor.py [hunk 1]
```diff
@@ -124,19 +126,42 @@ def get_user_email(self, egi_user):
             return f"{egi_user} not found in LDAP, has VO membership expired?"
         return self.user_emails[egi_user]
 
+    def get_public_ip(self, ip_addresses):
+        result = ""
+        for ip in ip_addresses:
+            if not ip.startswith("192.168") and not ip.startswith("172.16"):
```
Comment (enolfc) [inline fedcloud_vm_monitoring/site_monitor.py]: I suggest to use https://docs.python.org/3/library/ipaddress.html

Comment (enolfc) [reply fedcloud_vm_monitoring/site_monitor.py]: https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address.is_private

**Review by enolfc — CHANGES_REQUESTED** [review]
Please check comment

File: fedcloud_vm_monitoring/site_monitor.py (hunk 1 above)
Comment (sebastian-luna-valero) [reply fedcloud_vm_monitoring/site_monitor.py]: Great suggestion, many thanks!

**Review by enolfc — APPROVED** [review]
LGTM

**Pull request merged**